File: code1/fizzbuzz.java
/*
 * FizzBuzz
 * author: Ashwek
 * github: @ashwek
 */

class fizzbuzz{

	public static void fizzbuzz(int limit){

		for(int i=1; i<=limit; i++) {
	        if( i%15 == 0 )
	            System.out.println("FizzBuzz");
	        else if( i%3 == 0 )
	            System.out.println("Fizz");
	        else if( i%5 == 0 )
	            System.out.println("Buzz");
	        else
	            System.out.println(i);
	    }
	}

	public static void main(String args[]){
		fizzbuzz(100);
	}
}

File: code1/fizzbuzz.py
'''
* FIZZBUZZ
* author: Rajdeep Roy Chowdhury<rrajdeeproychowdhury@gmail.com> 
* github: https://github.com/razdeep
'''

def fizzbuzz(n):
    if n%5 == 0 and n%3==0:
        print('FizzBuzz')
    elif(n%5 == 0):
        print('Buzz')
    elif(n%3 == 0):
        print('Fizz')
    else:
        print(n)

if __name__=='__main__':
    for i in range(0,101):
        fizzbuzz(i)